frontend/src/types/pipeline/task.ts

```diff
@@ -112,13 +112,16 @@ export type TaskDatabasePITRDeletePayload = {
   // more input and output parameters in the future
 };
 
+export type RollbackSQLStatus = "PENDING" | "DONE" | "FAILED";
+
 export type TaskDatabaseDataUpdatePayload = {
   skipped: boolean;
   skippedReason: string;
   statement: string;
   sheetId: SheetId;
   pushEvent?: VCSPushEvent;
   rollbackEnabled: boolean;
+  rollbackSqlStatus?: RollbackSQLStatus;
```

Comment:
```suggestion
  rollbackSqlStatus?: RollbackSqlStatus;
```

Reply:
And maybe other places.

Reply:
Maybe not, since the type name is `RollbackSQLStatus`